### PR TITLE
Split

### PR DESCRIPTION
**Pull request by 754213Fishe**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] [local build the book](https://github.com/ocademy-ai/machine-learning/blob/main/CONTRIBUTING.md#build-the-book)
- [ ] review the artifacts from local build
- [ ] [local build the slides](https://github.com/ocademy-ai/machine-learning/blob/main/CONTRIBUTING.md#build-the-slides-optional) if appliable
- [ ] Unit test if appliable

# Checklist:

- [ ] My code follows the [style guidelines](https://github.com/ocademy-ai/machine-learning/blob/main/STYLE_GUIDE.md) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have applied tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: open-machine-learning-jupyter-book/data-science/working-with-data/Indexing_on_ndarrays.ipynb [hunk 1]
```diff
@@ -0,0 +1,2166 @@
+{
```
Comment (Lola-jo) [inline open-machine-learning-jupyter-book/data-science/working-with-data/Indexing_on_ndarrays.ipynb:1]: <p>A secondary heading cannot be used as the first title of a document.</p><p>The solution is to add a first-level heading in front of it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (Lola-jo) [inline open-machine-learning-jupyter-book/data-science/working-with-data/Indexing_on_ndarrays.ipynb:1]: <p>This code block is not working, try to fix it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (Lola-jo) [inline open-machine-learning-jupyter-book/data-science/working-with-data/Indexing_on_ndarrays.ipynb:1]: <p>This is a system error, not a block of code, remove it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

File: open-machine-learning-jupyter-book/data-science/working-with-data/Structured_arrays.ipynb [hunk 2]
```diff
@@ -0,0 +1,1912 @@
+{
```
Comment (Lola-jo) [inline open-machine-learning-jupyter-book/data-science/working-with-data/Structured_arrays.ipynb:1]: <p>A secondary heading cannot be used as the first title of a document.</p><p>The solution is to add a first-level heading in front of it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Comment by Lola-jo** [comment]
Please do not upload the rest of the files

File: open-machine-learning-jupyter-book/data-science/working-with-data/Indexing_on_ndarrays.ipynb (hunk 1 above)
Comment (754213Fishe) [reply open-machine-learning-jupyter-book/data-science/working-with-data/Indexing_on_ndarrays.ipynb:1]: So,can I just delete it?

Comment (754213Fishe) [reply open-machine-learning-jupyter-book/data-science/working-with-data/Indexing_on_ndarrays.ipynb:1]: <p>I figured out what was wrong with this block: the original md file wanted to give an example of code that reported an error out of bounds, so the teacher set it as code and actively asked it to report an error, and recorded the error content. I set it and its error reporting to markdown format, and no more errors were reported.</p><p>And I delete the redundant files</p>

**Pull request closed without merging**